app/assets/stylesheets/components/syntax.scss

```diff
@@ -68,6 +81,9 @@ div.highlight {
   &:hover {
     opacity: 1;
   }
+  &:focus {
+    outline: none;
```

Comment:
@aitchiss, I imagine we'd still need some kind of visual indicator that this has focus or stay with the default via focus-within.